etl/src/replication/table_sync.rs

```diff
@@ -260,14 +249,19 @@ where
     // We mark this worker as `SyncWait` (in memory only) to signal the apply worker that we are
     // ready to start catchup.
     {
-        let mut inner = table_sync_worker_state.get_inner().lock().await;
+        let mut inner = table_sync_worker_state.lock().await;
         inner
-            .set_phase_with(TableReplicationPhase::SyncWait, state_store)
+            .set_and_store(TableReplicationPhase::SyncWait, &state_store)
             .await?;
 
         // We notify the main apply worker to force syncing tables. In this way, the `Catchup` phase
         // will be started even if no events are flowing in the main apply loop.
-        let _ = force_syncing_tables_tx.send(());
+        if force_syncing_tables_tx.send(()).is_err() {
+            error!(
```

Comment:
Added the error to make this error visible, we can't do much if there is a failure here.